mediaqueries/Overview.bs

```diff
@@ -1597,18 +1597,24 @@ Interaction Media Features</h2>
 		</style>
 	</div>
 
-	The 'pointer' and 'hover' features relate to the characteristics of the “primary” input mechanism, 
-	while 'any-pointer' and 'any-hover' can be used to query the properties of all potentially available input mechanisms.
+	The 'pointer' and 'hover' features relate to the characteristics of the “primary” pointing device, 
+	while 'any-pointer' and 'any-hover' can be used to query the properties of all potentially available pointing devices.
 
-	Note: While this specification does not define how User Agents should decide what the “primary” input is, 
+	Note: While this specification does not define how User Agents should decide what the “primary” pointing device is, 
 	the expectation is that User Agents should make this determination 
 	by combining knowledge about the device/environment they are running on, 
-	the number and type of input mechanisms available, 
-	and a notion of which of these inputs is generally and/or currently being used. 
-	User Agents may also decide to dynamically change what type of input is deemed to be primary, 
+	the number and type of pointing devices available, 
+	and a notion of which of these is generally and/or currently being used. 
+	User Agents may also decide to dynamically change what type of pointing device is deemed to be primary, 
```

Comment:
could add an extra example right here: "In situations where the primary input mechanism for a device is not a pointing device, but there is a secondary – and less frequently used –  input that is a pointing devices, the user agent may decide to keep treating the non-pointing device as the primary (resulting in `pointer: none`), or they may opt to "promote" the secondary input to be identified as the primary pointing device (resulting in `pointer: fine` or `pointer: coarse`).